mondey_backend/src/mondey_backend/routers/users.py

```diff
@@ -187,4 +198,65 @@ def update_current_user_answers(
         session.commit()
         return new_answers
 
+    # Endpoints for answers to child question
+    @router.get("/children-answers/{child_id}", response_model=list[ChildAnswerPublic])
+    def get_current_child_answers(
+        session: SessionDep, child_id: int, current_active_user: CurrentActiveUserDep
+    ):
+        child = session.get(
+            Child,
+            child_id,
+        )
+
+        if child is None or child.user_id != current_active_user.id:
+            raise HTTPException(404, detail="child not found for user")
+        answers = session.exec(
+            select(ChildAnswer).where(
+                (col(ChildAnswer.user_id) == current_active_user.id)
+                & (col(ChildAnswer.child_id) == child_id)
+            )
+        ).all()
+
+        return answers
+
+    @router.put("/children-answers/{child_id}", response_model=list[ChildAnswerPublic])
+    def update_current_child_answers(
+        session: SessionDep,
+        child_id: int,
+        current_active_user: CurrentActiveUserDep,
+        new_answers: list[ChildAnswerPublic],
+    ):
+        child = session.get(
+            Child,
+            child_id,
+        )
+
+        # because child and childanswers have overlap, we need to keep both in sync
+        if child is None or child.user_id != current_active_user.id:
+            raise HTTPException(404, detail="child not found for user")
+
+        for new_answer in new_answers:
+            current_answer = session.get(
+                ChildAnswer, (current_active_user.id, child_id, new_answer.question_id)
+            )
+
+            if current_answer is None:
+                current_answer = ChildAnswer.model_validate(
+                    new_answer,
+                    update={
+                        "user_id": current_active_user.id,
+                        "child_id": child_id,
+                        "question_id": new_answer.question_id,
+                    },
+                )
+            else:
+                for key, value in new_answer.model_dump().items():
+                    setattr(current_answer, key, value)
+
+            add(session, current_answer)
+        add(session, child)
+        session.commit()
+
+        return new_answers
```

Comment:
I realise this is following what I did previously, but looking at it again I don't think it makes much sense!
Here we just return whatever we got sent which seems pointless - I think it would make more sense to just `return {"ok": True}`.
The alternative would be to return the new set of answers from the database, but if it worked then they should anyway match what we got sent, so I think just returning ok would make the most sense here.